rbac/internal/utils.py

```diff
@@ -117,3 +119,62 @@ def get_or_create_ungrouped_workspace(tenant: str) -> Workspace:
             workspace, ReplicationEventType.CREATE_WORKSPACE
         ).replicate_new_workspace()
     return workspace
+
+
+def validate_relations_input(request, request_data) -> bool:
+    """Check if request body provided to relations tool endpoints are valid."""
+    valid_endpoints = {"/_private/api/relations/lookup_resource/"}
+    request_endpoint = request.path
+    request_method = request.method
+
+    if request_endpoint in valid_endpoints and request_method == "POST":
+        match request_endpoint:
+            case "/_private/api/relations/lookup_resource/":
+                schema = {
```

Comment:
this schema definition can go into separated file and into constant/variable, where it might be beneficial  to name constant/variable by request (something like `LOOKUP_RESOURCE_INPUT_SCHEMA`)